### ISSUE TITLE
sdist on PyPI contains a lot of stray files

### ISSUE THREAD

**Issue by mgorny** [wimglenn/pytest-freezer#9]
`pytest_freezer-0.4.6.tar.gz` contains:

```
pytest_freezer-0.4.6/.coverage
pytest_freezer-0.4.6/.idea/*
pytest_freezer-0.4.6/.pytest_cache/**
pytest_freezer-0.4.6/.venv/**
```

All of them seem to be temporary files that weren't intended to end up in there.

**Comment by wimglenn** [comment]
Gosh, you're right, an [8MB sdist](https://pypi.org/project/pytest-freezer/0.4.6/#files)! It was my first time using hatch and guess I just assumed it would exclude all that kind of stuff by default.

Fixed in [0.4.7](https://pypi.org/project/pytest-freezer/#files), thanks for the notice.

**Comment by mgorny** [comment]
No problem. Thanks!
